enhancements/etcd/cluster-etcd-operator.md

```diff
@@ -180,6 +180,16 @@ This controller shapes the static pod manifest itself.  It has some unique featu
     It waits until it is able to know that it is in the member list before trying to start.
     When it does this, it is able to determine the member list to use to launch.
 
+### Replacing single lost master
+This is one of the scenarios that needs user intervention and the step needs to be in user-facing docs.
+
+1. User will determine if one of the etcd is unhealthy by verifying against etcd API from etcd pod: `etcdctl endpoint health --cluster`
```

Comment:
This confuses the idea of "member gone bad" with "master host is gone".  Figure out which one you're working to fix, write that in the top level summary, and then re-focus this doc.  You need a pre-condition check.  For instance, "confirm that your host is gone and will not come back" and "now confirm using the etcd member list that the expected member is unhealthy"